docs/excel/custom-functions-overview.md

```diff
@@ -299,7 +299,7 @@ As you can see, ranges are handled in JavaScript as arrays of row arrays (like a
 
 - **Nov 7, 2017**: Shipped* the custom functions preview and samples
 - **Nov 20, 2017**: Fixed compatibility bug for those using builds 8801 and later
-- **Nov 28, 2017**: Shipped support* for cancellation on asynchronous functions (requires change for streaming functions)
-- **May 7, 2018**: Shipped support* for Mac, Excel Online, and synchronous functions running in-process
+- **Nov 28, 2017**: Shipped* support for cancellation on asynchronous functions (requires change for streaming functions)
+- **May 7, 2018**: Shipped* support for Mac, Excel Online, and synchronous functions running in-process
 
-* - to the Office Insiders Channel
+/* to the Office Insiders Channel
```

Comment:
I think this should be a backslash `\` (otherwise the slash [appears on the site](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=keyur32-patch-1)).

Reply:
Yep, of course!  Just noticed that when reviewing the change on the Staging site...thanks!